src/core/pluralize.js

```diff
@@ -0,0 +1,68 @@
+// Adds automatic pluralization to dfns
+// If a dfn is referenced as it's plural (and plural of `data-lt` attributes),
+//   plurals of it are automatically added to `data-plurals`.
+// The linking is done in core/link-to-dfn
+
+import { plural as pluralOf } from "deps/pluralize";
+import { norm as normalize } from "core/utils";
+
+export const name = "core/pluralize";
+
+export function run(conf) {
+  if (!conf.pluralize) return;
+
+  if (!conf.hasOwnProperty("definitionMap")) {
```

Comment:
we should probably move this now to base-runner.js, because we are doing this check twice now.

In base runner, we want:

```JS
const respecDefaults = {
  conf.definitionMap: Object.create(null);
}

// and in runAll()
const conf = { ...respecDefaults, ...window.respecConfig };
```

Reply:
fixed typo.

Reply:
And then we can remove this from check from src/core/dfn.js